ip/rest_resolver.go

```diff
@@ -11,15 +11,15 @@ import (
 	"time"
 )
 
-const IPIFY_API_URL = "https://api.ipify.org/"
-const IPIFY_API_CLIENT = "goclient-v0.1"
-const IPIFY_API_LOG_PREFIX = "[ipify.api] "
+const IpifyAPIURL = "https://api.ipify.org/"
+const IpifiAPICLient = "goclient-v0.1"
```

Comment:
`Ipifi` -> `Ipify..`
+ Could be private constants

Reply:
Done.